[2024-06-11T22:56:07.117+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task4 manual__2024-06-11T22:56:01.032040+00:00 [queued]>
[2024-06-11T22:56:07.121+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: example_dag_with_callback.task4 manual__2024-06-11T22:56:01.032040+00:00 [queued]>
[2024-06-11T22:56:07.122+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T22:56:07.122+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-06-11T22:56:07.122+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-06-11T22:56:07.128+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): task4> on 2024-06-11 22:56:01.032040+00:00
[2024-06-11T22:56:07.131+0000] {standard_task_runner.py:55} INFO - Started process 4611 to run task
[2024-06-11T22:56:07.134+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'example_dag_with_callback', 'task4', 'manual__2024-06-11T22:56:01.032040+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/example_dag_with_callback.py', '--cfg-path', '/tmp/tmpi9l2vo52']
[2024-06-11T22:56:07.134+0000] {standard_task_runner.py:83} INFO - Job 252: Subtask task4
[2024-06-11T22:56:07.165+0000] {task_command.py:376} INFO - Running <TaskInstance: example_dag_with_callback.task4 manual__2024-06-11T22:56:01.032040+00:00 [running]> on host 91b1d49ba724
[2024-06-11T22:56:07.197+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_dag_with_callback
AIRFLOW_CTX_TASK_ID=task4
AIRFLOW_CTX_EXECUTION_DATE=2024-06-11T22:56:01.032040+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-06-11T22:56:01.032040+00:00
[2024-06-11T22:56:07.197+0000] {logging_mixin.py:120} INFO - Task 4 is completed
[2024-06-11T22:56:08.201+0000] {python.py:177} INFO - Done. Returned value was: None
[2024-06-11T22:56:08.226+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=example_dag_with_callback, task_id=task4, execution_date=20240611T225601, start_date=20240611T225607, end_date=20240611T225608
[2024-06-11T22:56:08.249+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-06-11T22:56:08.259+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
